scout/ecm_prep.py

```diff
@@ -9582,8 +9677,8 @@ def partition_microsegment(
         # Return partitioned stock, energy, and cost mseg information
         return [stock_total, energy_total, carb_total, fmeth_total,
                 frefr_total, stock_total_meas, energy_total_eff,
-                carb_total_eff, fmeth_total_eff, frefr_total_eff,
-                stock_compete, energy_compete, carb_compete,
+                energy_total_eff_capt, carb_total_eff, fmeth_total_eff,
+                frefr_total_eff, stock_compete, energy_compete, carb_compete,
```

Comment:
@jtlangevin Is the below a good "Returns" docstring for partition_microsegment?

        Returns:
            By year, total, total-efficient, competed, and competed-efficient
            stock, energy, carbon, and cost market microsegments; methane and
            refrigerant leakage fugitive emissions. For fuel switching measures,
            also return any remaining (unswitched) energy, carbon, and cost
            segments by year.

Reply:
That looks good to me @trynthink.

Reply:
@aspeake yes I think adding `eff_capt` as a user argument might get confusing given its use in the code. If we were to invert it in this way, we could name it something different, like, `eff_capt_report` to avoid confusion. Doing so would just require that we adjust the tests, which are generally designed around cases where it is _not_ reported (e.g., it would be False).